resources/assets/components/Tags/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class Tags extends React.Component {
+  render() {
+
+    return (
+        <ul className="form-actions -inline">
+          <li><b> Tags </b></li>
+          <br/>
+          <li> <input className="button -secondary" value="Good Photo"/></li>
```

Comment:
When we end up displaying reviewed reportbacks in the admin interface, we'd need to be able to pass in values for each of these tags (e.g. `{'good_photo': true, 'good_quote': false}`).

It'd be nice to not have to manually check `values['good_photo']` and `values['good_quote']` and so on in order to render out the state of the interface.